builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md

```diff
@@ -25,6 +25,23 @@ The demo EVM ContainerChain HTTPS and WSS endpoints are as follows:
     wss://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network/
     ```
 
+## Faucet {: #faucet }
+
+You can access TANGO tokens at the faucet at [apps.tanssi.network/demo](https://apps.tanssi.network/demo){target=\_blank}. You can receive up to 100 TANGO tokens every 12 hours. Please do not spam the faucet with unnecessary requests. To request tokens from the faucet, head to [apps.tanssi.network/demo](https://apps.tanssi.network/demo){target=\_blank} and Press **Add to MetaMask**.
```

Comment:
```suggestion
You can access TANGO tokens at the faucet on the [Tanssi dApp](https://apps.tanssi.network/demo){target=\_blank}. You can receive up to 100 TANGO tokens every 12 hours. Please do not spam the faucet with unnecessary requests. To request tokens from the faucet, head to the [Tanssi dApp](https://apps.tanssi.network/demo){target=\_blank} and Press **Add to MetaMask**.
```

This actually came up in a recent report, but using the link as the link text is not considered descriptive, so just saying something like "Tanssi dApp" would pass that check

Reply:
you could also say "Tanssi faucet" here... or whatever you think is best!